docs/build.md

```diff
@@ -0,0 +1,405 @@
+# Building from the ground up
```

Comment:
We have many steps and sub steps to build a dcos-net cluster, do you mind adding subtitles to make it more organized, if it can?

## preparation
## compile erlang
## compile DC/OS Mesos module
## start up the components
Exhibitor, Mesos-master and Mesos-DNS are deployed on master node, and Mesos-agent are deployed on agent node, and dcos-net runs on both nodes.
### start Exhibitor (master node)
### start Mesos-master (master node)
### start Mesos-DNS (master node)
### start Mesos-agent (agent node)
### start dcos-net
1.  ...
2.  ...